concepts/05 Widgets/TileView/05 Customize Tile Appearance.md

```diff
@@ -1,7 +1,7 @@
 For a minor customization of tiles, you can define [specific fields](/api-reference/10%20UI%20Widgets/dxTileView/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxTileView/Configuration/items/') in item data objects. For example, the following code generates three tiles: the first is disabled, the second is not customized, the third is hidden.
 
 ---
-#####jQuery
+##### jQuery
```

Comment:
Please make the same changes that were made to a similar file for the [Tabs](https://github.com/DevExpress/devextreme-documentation/pull/329/files#diff-73a7b01724c5a539d83ef67c507f494fR3) widget.

Reply:
fixed